packages/expo-updates/ios/EXUpdates/UpdatesError.swift

```diff
@@ -0,0 +1,109 @@
+//  Copyright © 2021 650 Industries. All rights reserved.
+
+import Foundation
+
+internal enum UpdatesError: LocalizedError {
```

Comment:
We should maybe think about consolidating our different error enums -- we have this one, another one for the logger, etc.